packages/webservice/app/src/main/kotlin/com/yahoo/navi/ws/config/ElideConfig.kt

```diff
@@ -29,4 +31,9 @@ class ElideConfig {
             .withISO8601Dates("yyyy-MM-dd HH:mm:ss", TimeZone.getTimeZone("UTC"))
         return Elide(builder.build())
     }
+
+    @Bean
+    fun buildAsyncQueryDAO(elide: Elide): AsyncQueryDAO? {
```

Comment:
This will cause errors when the AsyncCleanerService is executed. There is a fix being worked upon for this in Elide.